main.py

```diff
@@ -1,5 +1,55 @@
 import init_django_orm  # noqa: F401
+from db.models import Genre, Actor
 
 
 def main():
-    pass
+    create()
+    update()
+    delete()
+
+    return Actor.objects.filter(last_name="Smith").order_by("first_name")
```

Comment:
It's a shame that this `Retrieve` step here is not complicated enough for you to decide it's worth extracting a function. :)